README.md

```diff
@@ -257,6 +257,21 @@ could be used like this:
 .isIE .foo { color: red; }
 ```
 
+### Comma-separated CSS values
+
+In order to pass a comma-separated value as a argument to a mixin, you can use
+the special `asSingleArg` keyword. For example:
```

Comment:
Other names that I came up with while coding:
- `single-arg`
- `mixinArg`
- `mixin-arg`

Reply:
I think CSS code style is more about `single-arg`

camelCase is very rare.